### PR TITLE
Update ReadMe

### PR DESCRIPTION
**Pull request by CosmicShreyas**
Fixed some command issues, added model links and updated the Image

### CODE REVIEW

**Comment by Shreyas-ITB** [comment]
Well new members who are new to AI's and stuff doesn't know where to download the models. Even i had the problem. And im not hijacking the links tho. HuggingFace is opensource im just getting the links **with the permission of the owner** and pasting it there in the readme file. and its not even direct link to download the model, if you click on one of those links it just shows the files of the model that if you click on will download the file.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
 # llama.cpp
 
-![llama](https://user-images.githubusercontent.com/1991296/227761327-6d83e30e-2200-41a6-bfbb-f575231c54f4.png)
+![https://raw.githubusercontent.com/Shreyas-ITB/-/main/Friendly-Alpaca.png](https://raw.githubusercontent.com/Shreyas-ITB/---/main/Friendly-Alpaca.png)
```
[the remark this answers is not in the document]
Comment (Shreyas-ITB) [reply README.md:3]: That logo was very cool so i thought of changing it, if you guys dont like it then u can revert the logo change back.

[the remark this answers is not in the document]
Comment (Shreyas-ITB) [reply README.md:3]: use the second link [this one](https://raw.githubusercontent.com/Shreyas-ITB/---/main/Friendly-Alpaca.png)

**Comment by prusnak** [comment]
> And im not hijacking the links tho.

You changed the logo with no discussion.
You changed 3 links from https://github.com/ggerganov to https://github.com/Shreyas-ITB

👎

**Comment by Shreyas-ITB** [comment]
Again guys if you dont want this pr just close it off. I dont mind it 👍

**Pull request closed without merging**